packages/api-sdk/src/resources/projectSecret.ts

```diff
@@ -0,0 +1,38 @@
+import type Fetch from '@api-sdk/fetch';
+import { BaseModels } from '@voiceflow/base-types';
+
+import BaseResource from './base';
+
+const ENDPOINT = 'projects';
+
+class ProjectSecretResource extends BaseResource<ProjectSecretResource> {
+  constructor(fetch: Fetch) {
+    super({ fetch, clazz: ProjectSecretResource, endpoint: ENDPOINT });
+  }
+
+  public async findByProjectID(projectID: string, tag: number): Promise<string> {
+    const { data } = await this.fetch.get<string>(`${this.endpoint}/${projectID}/secret/${tag}`);
+
+    return data;
+  }
+
+  public async findByLookup(lookup: string, tag: number): Promise<BaseModels.ProjectSecret.Model[]> {
+    const { data } = await this.fetch.get<BaseModels.ProjectSecret.Model[]>(
+      `${this.endpoint}/secret?lookup=${encodeURIComponent(lookup)}&tag=${tag}`
+    );
+
+    return data;
+  }
```

Comment:
I'm also curious, how come for one of these, you return only the secret, but for the other, you return the entire ProjectSecret object?

Reply:
you are right, nice catch!